src/PlainTextSummaryBuilder.php

```diff
@@ -167,6 +167,9 @@ public function appendAvailability(Status $status, BookingAvailability $bookingA
                 $c->workingLine[] = '(' . $this->translator->translate('postponed') . ')';
                 return $c;
             default:
+                if ($bookingAvailability->getType() === 'Unavailable') {
```

Comment:
I prefer encapsulating these checks with a dedicated method like this:
```
if ($bookingAvailability->isUnavailable()) { ... }
```

Reply:
Added `isAvailable` convenience method